core/contracts/financial-templates/implementation/FeePayer.sol

```diff
@@ -63,9 +63,9 @@ abstract contract FeePayer is Testable {
      * @notice Constructs the FeePayer contract. Called by child contracts.
      * @param collateralAddress ERC20 token that is used as the underlying collateral for the synthetic.
      * @param finderAddress UMA protocol Finder used to discover other protocol contracts.
-     * @param isTest whether this contract is being constructed for the purpose of running tests.
+     * @param timerAddress Universal store of time for contracts in test environment.
```

Comment:
nit: I would add that this parameter should be set to 0x0 for production environments, so they will fall back to block time. Here and elsewhere.

Reply:
Good call. Changed to:

```
     * @param _timerAddress Contract that stores the current time in a testing environment. Should be set to 0x0 for production environments that use live time.
```

Reply:
One very tiny nit. I would change *should* to *must*.